app/controllers/users/confirmations_controller.rb

```diff
@@ -2,6 +2,12 @@ module Users
   class ConfirmationsController < Devise::ConfirmationsController
     include ValidEmailParameter
 
+    def create
+      super
+
+      UserMailer.already_confirmed(resource).deliver_later if resource.confirmed?
```

Comment:
I guess it wasn't completely clear to me how the user might end up at the #create action, but thats probably because I'm new to rails.